tsfile/src/main/java/org/apache/iotdb/tsfile/encoding/bitpacking/LongPacker.java

```diff
@@ -161,11 +154,10 @@ public void unpack8Values(byte[] buf, int offset, long[] values) {
    * array named 'values'.
    *
    * @param buf array where all bytes are in.
-   * @param offset the offset of first byte to be decoded in buf.
    * @param length length of bytes to be decoded in buf.
    * @param values decoded result
    */
-  public void unpackAllValues(byte[] buf, int offset, int length, long[] values) {
+  public void unpackAllValues(byte[] buf, int length, Long[] values) {
```

Comment:
Why Long[]?

Reply:
These kinds of modifications will be revoked.